scripts/deploy.sh

```diff
@@ -7,4 +7,6 @@ which ruby
 ruby -v
 bundle exec rake deploy
 
+git config --global user.email travis@holidayextras.com
```

Comment:
Might be to late setting it here - set it as early as possible?

Reply:
Ok